pkg/osquery/interactive/interactive_test.go

```diff
@@ -1,3 +1,4 @@
+// disabled on windows because go tests doesn't get access to used named pipes
```

Comment:
Really!?

Reply:
@directionless , I looked into this deeper and this turned out to be an issue with the tests trying to download the osqueryd binary in parallel to the same location. I moved the initial download and caching of the osqueryd binary to the TestMain func and it cleared up the issue.

Reply:
@directionless , nevermind, I forgot to call m.Run() at the end of test main .... so the tests didnt run ... back to the drawing board.